processing/src/main/java/io/druid/segment/QueryableIndexStorageAdapter.java

```diff
@@ -1055,6 +1089,20 @@ public boolean matches()
         }
       };
     }
+
+    private ValueMatcher makeLongValueMatcher(String dimension, final DruidLongPredicate predicate)
```

Comment:
@gianm probably not absolutely needed, but I wanted to have a method for each type, for consistency reasons